imagesgallery/imagesgallery.py

```diff
@@ -74,22 +96,56 @@ def student_view(self, context=None):
         if context:
             pass  # TO-DO: do something based on the context.
 
-        # Remove the bundle default generated by webpack
-        script_to_remove = r'<script defer="defer" src="bundle.js"></script>'
+        xblock_id = self.block_id
+        # XBlock IDs typically follow this format: block-v1:{course_name}+type@{xblockname}+block@{id_xblock}
+        # This code extracts the last part of the string, which is {id_xblock}
+        xblock_id_splitted = self.block_id_parsed
```

Comment:
Can you delete this variable and directly access the attribute where you need it?
Same for the `block_id` variable above

Reply:
Yep, done